src/latency.c

```diff
@@ -279,7 +279,7 @@ sds createLatencyReport(void) {
 
         /* Potentially commands. */
         if (!strcasecmp(event,"command")) {
-            if (server.slowlog_log_slower_than < 0) {
+            if (server.slowlog_log_slower_than < 0 || server.slowlog_max_len == 0) {
```

Comment:
We presumably also have the same issue for the ACL log.

Reply:
Yeah, mentioned in the top comment.